blocks/init/src/Blocks/custom/tabs/assets/index.js

```diff
@@ -0,0 +1,24 @@
+import domReady from '@wordpress/dom-ready';
+import {blockJsClass, tabJsClass, tabPanelJsClass, tabVisibleClass, tabPanelItemActiveClass, tabPanelItemTemplateId, tabPanelItemButtonJsClass} from './../manifest.json'
+
+domReady(() => {
+    const tabSelector = `.${blockJsClass}`;
+    const tabElements = document.querySelectorAll(tabSelector);
+
+    if(tabElements.length){
+        import('./tabber').then(({ Tabber }) => {
+			[...tabElements].forEach((item) => {
+				const tabber = new Tabber({
+                    tabsElement: item,
+                    tabJsClass,
+                    tabPanelJsClass,
+                    tabVisibleClass,
+                    tabPanelItemActiveClass,
+                    tabPanelItemTemplateId,
+                    tabPanelItemButtonJsClass
+                });
+				tabber.init();
+			});
+		});
```

Comment:
You're instantiating the `Tabber` class inside a loop. This is a no-no 😄 

You should pass the `tabElements` to the class, alongside all the options (you can extract them out as a variable and then pass to the class like `new Tabber(options)`).

Then inside the `Tabber` class, you'd have the logic for handling all the elements (foreaching and whatnot) 🙂 

The same as you'd do in PHP - you need to be mindful when looping things.